letsencrypt/client/crypto_util.py

```diff
@@ -124,68 +129,35 @@ def make_ss_cert(key_str, domains, not_before=None,
     pubkey = M2Crypto.EVP.PKey()
     pubkey.assign_rsa(rsa_key)
 
-    cert = M2Crypto.X509.X509()
-    cert.set_pubkey(pubkey)
-    cert.set_serial_number(1337)
-    cert.set_version(2)
+    m2_cert = M2Crypto.X509.X509()
```

Comment:
Yes, this was an artifact of when get_cert_info(cert) was still in the file.  I thought it made it more confusing to have two different types of certs in the same file.

I will change it back though.